src/datetime/mod.rs

```diff
@@ -1247,6 +1247,16 @@ impl<Tz: TimeZone> AddAssign<Duration> for DateTime<Tz> {
     }
 }
 
+impl<Tz: TimeZone> Add<FixedOffset> for DateTime<Tz> {
```

Comment:
In my opinion this impl doesn't make much sense. Should the offset be added to the datetime, or to the offset? And why? I would like to remove it on main.

Reply:
Removing it on main sounds good to me.

Reply:
I'll make a PR soon-ish.